cluster/engine.go

```diff
@@ -1015,12 +1015,28 @@ func (e *Engine) Pull(image string, authConfig *types.AuthConfig) error {
 
 // Load an image on the engine
 func (e *Engine) Load(reader io.Reader) error {
-	err := e.client.LoadImage(reader)
+	loadResponse, err := e.apiClient.ImageLoad(context.TODO(), reader, false)
```

Comment:
We can leave it as is for now until the integration with engine-api is over and we determine where the use of a context is critical, but we should use the contexts properly deriving a default context and adding timeouts when necessary or cancel the context under exceptional circumstances. There are no guarantee that the call to `ImageLoad` will ever return. And the empty context does not capture a timeout value.